.github/workflows/basic-tests.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         cd lalsuite-extra
         git lfs pull -I "data/lalsimulation/SEOBNRv2ROM_*.dat"
         git lfs pull -I "data/lalsimulation/*ChirpTime*.dat"
-        git lfs pull -I "data/lalsimulation/SEOBNRv4ROM_v2.0.hdf5"
+        git lfs pull -I "data/lalsimulation/SEOBNRv4ROM_v*.hdf5"
```

Comment:
Curious about this. Looking at the Gitlab repo, [that file](https://git.ligo.org/lscsoft/lalsuite-extra/-/blob/master/data/lalsimulation/SEOBNRv4ROM_v2.0.hdf5) has not been changed in 8 years (!).

Reply:
Yup, looking deeper in the errors, it seems to be wanting v3.0 of that file, and it doesn't exist that I can see

Reply:
In fact it could be [this change](https://git.ligo.org/lscsoft/lalsuite/-/commit/bcb1dabf2c3f2c4e761a0967ae5e68ab9a2a5d3f) that caused the error

Reply:
Wait, is this on a lalsuite release? How did that slip through??

Reply:
Blocking the latest lalsuite version is a viable workaround (latest tests), but I know that we generally don't like blocking versions of dependencies.

It is not planned for the required files to be on lalsuite-extra, so we will need to access an alternative.

I think blocking that version for now until the new location is decided, and then we can unblock it once we point to the new location

Reply:
Pinning lalsuite to *less* than this version might be needed temporarily (don't block the release, pin to less than)